bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/NMSSM_XYH_YTobb_HToWWTo2QLNu/NMSSM_XYH_YTobb_HToWWTo2QLNu_MX_mMX_MY_mMY_run_card.dat

```diff
@@ -0,0 +1,271 @@
+  average =  event_norm       ! average/sum. Normalization of the weight in the LHEF
+
+#*********************************************************************
+# Matching parameter (MLM only)
+#*********************************************************************
+ 0 = ickkw            ! 0 no matching, 1 MLM
+ 1.0 = alpsfact         ! scale factor for QCD emission vx
+ False = chcluster        ! cluster only according to channel diag
+ 4 = asrwgtflavor     ! highest quark flavor for a_s reweight
+ False  = auto_ptj_mjj  ! Automatic setting of ptj and mjj if xqcut >0
+                                   ! (turn off for VBF and single top processes) 
+ 0.0   = xqcut   ! minimum kt jet measure between partons
+#*********************************************************************
+#
+#*********************************************************************
+# handling of the helicities:
+#  0: sum over all helicities
+#  1: importance sampling over helicities
+#*********************************************************************
+   0  = nhel          ! using helicities importance sampling or not.
+#*********************************************************************
+# Generation bias, check the wiki page below for more information:   *
+#  'cp3.irmp.ucl.ac.be/projects/madgraph/wiki/LOEventGenerationBias' *
+#*********************************************************************
+ None = bias_module  ! Bias type of bias, [None, ptj_bias, -custom_folder-]
+ {} = bias_parameters ! Specifies the parameters of the module.
+#
+#*******************************                                                 
+# Parton level cuts definition *
+#*******************************                                     
+#                                                                    
+#
+#*********************************************************************
+# BW cutoff (M+/-bwcutoff*Gamma) ! Define on/off-shell for "$" and decay  
+#*********************************************************************
+  15.0  = bwcutoff      ! (M+/-bwcutoff*Gamma)
+#*********************************************************************
+# Apply pt/E/eta/dr/mij/kt_durham cuts on decay products or not
+# (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
+#*********************************************************************
+  True  = cut_decays    ! Cut decay products 
+#*********************************************************************
+# Standard Cuts                                                      *
+#*********************************************************************
+# Minimum and maximum pt's (for max, -1 means no cut)                *
+#*********************************************************************
+ 20.0  = ptj       ! minimum pt for the jets 
+ 0.0  = ptb       ! minimum pt for the b 
+ 0.0  = pta       ! minimum pt for the photons 
+ 0.0  = ptl       ! minimum pt for the charged leptons 
```

Comment:
is it on purpose to cut on jet in decay but not the others? if not intended i suggest to set the cut to 0.

Reply:
We always use jet pt cut of 30 GeV at the detector level, so 20 GeV cut in run card should be fine. Thresholds on lepton pt in offline selection is ~20 GeV, so we can also put a cut of 10 GeV here. For b quark jets, since we target to capture b quark pairs using AK8 jets (i.e. boosted topology), we didn't put any cut on individual b quark jets, but practically we can put a cut of 20 GeV here too. We don't put any cut on photons since we don't have those in our event topology.

Reply:
These are on purpose. We always use |eta|<2.5 on leptons at the reconstruction level, hence a loose cut of 3 here. For jets, we relaxed the cut more since we might use jets with |eta| < 3 in the analysis, but this can also be tightened (say, to 4).

Reply:
Width for the SM-like Higgs boson (pdg Id: 25) is  kept to the SM value of Higgs boson width, which is 4 MeV. For other heavy Higgs bosons  (pdg Id: 35 and 45), they are taken to be of narrow width; however the  choice of 1 MeV is arbitrary, but this is followed in similar X->YH analyses (ex: https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/NMSSM_XYH_bbbb/Template/customizecards.dat, https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/NMSSM_XYH_WWWW/Template/customizecards.dat).  Therefore, we would also like to keep it 1 MeV so that it's easier to combine the results with the analyses pointed above.